graphql/schema.graphql

```diff
@@ -1016,7 +1018,9 @@ type BaseTaskInfo {
 }
 
 type GroupedProjects {
-  name: String!
+  groupDisplayName: String! 
+  name: String @deprecated(reason: "name is deprecated. Use groupDisplayName instead.")
```

Comment:
can you re add the `!` since that field should always be present.

Reply:
sure